docs/references/index.md

```diff
@@ -10,5 +10,5 @@ DataChain's API is organized into several modules:
 - [DataType](./datatype.md) - Type system and schema definitions
 - [File](./file.md) - File handling and storage operations
 - [UDF](./udf.md) - User-defined functions and transformations
-- [SQL](./sql.md) - SQL query integration
+- [Func](./func.md) - SQL query integration
```

Comment:
Same thing here, need to remove mention of `SQL`.

Reply:
What would be the better name here. Utility functions ?

Reply:
let's just use Functions for now